x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/lib/action_connector_api.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HttpServiceBase } from 'kibana/public';
+import { BASE_ACTION_API_PATH } from '../constants';
+import { ActionConnector, ActionConnectorWithoutId, ActionType } from '../../types';
+
+// We are assuming there won't be many actions. This is why we will load
+// all the actions in advance and assume the total count to not go over 100 or so.
```

Comment:
Should we add this to telemetry so we can actually see if this assumption is correct?

Reply:
yes, we do. But this is a typo - it means connectors, not actions. I suppose that actions will be much more instances then connectors